hydra/core/singleton.py

```diff
@@ -18,12 +18,21 @@ def instance(cls: Any, *args: Any, **kwargs: Any) -> Any:
 
     @staticmethod
     def get_state() -> Any:
+        instances = deepcopy(Singleton._instances)
+        # Plugins can cause issues for pickling Singleton.get_state().
+        from hydra.core.plugins import Plugins
+
+        instances.pop(Plugins, None)
```

Comment:
```suggestion
        instances = deepcopy(Singleton._instances)
```suggestion
        # Plugins can cause issues for pickling the singleton state
        # Exclude them and re-initialize them on set_state()
        from hydra.core.plugins import Plugins

        instances.pop(Plugins, None)
```